ApplicationLibCode/Application/Tools/Summary/RiaSummaryStringTools.cpp

```diff
@@ -121,11 +121,12 @@ void RiaSummaryStringTools::splitUsingDataSourceNames( const QStringList& filter
 
         bool foundDataSource = false;
 
-        QRegExp searcher( pureDataSourceCandidate, Qt::CaseInsensitive, QRegExp::WildcardUnix );
+        QString            regexPattern = QRegularExpression::wildcardToRegularExpression( pureDataSourceCandidate );
+        QRegularExpression searcher( regexPattern, QRegularExpression::CaseInsensitiveOption );
 
         for ( const auto& ds : dataSourceNames )
         {
-            if ( !foundDataSource && searcher.exactMatch( ds ) )
+            if ( !foundDataSource && searcher.match( ds ).hasMatch() )
```

Comment:
Are you sure this is the same as `exactMatch`? https://doc.qt.io/qt-6/qregexp.html#exactMatch
Maybe also check `match.capturedLength(0) == ds.length())` to get similar behavior?

Reply:
Covered by unit test RiaSummaryStringToolsTest, works as expected